uint8 mode
uint8 MODE_STANDBY=0
uint8 MODE_FREEDRIVE=1
# Right now have only added modes that don't require setpoints
# Because modes requiring a setpoint should not be entered manually (should be entered implicitly using setpoints or a trajectory message instead)
